checks/runchecks.go

```diff
@@ -94,13 +104,90 @@ func RunChecks(ctx *context.Context) ([]*pkg.CheckResult, error) {
 		result := c.Run(ctx)
 		transformedResults := TransformResults(ctx, result)
 		results = append(results, transformedResults...)
-
 		ExportCheckMetrics(ctx, transformedResults)
 	}
 
+	if err := saveArtifacts(ctx, results); err != nil {
+		logger.Errorf("error saving artifacts: %v", err)
+	}
+
 	return ProcessResults(ctx, results), nil
 }
 
+func saveArtifacts(ctx *context.Context, results pkg.Results) error {
```

Comment:
Can we extract this function to an artifacts.go file thay we can just copy and paste for incident commander ?